libtransmission/announcer-udp.cc

```diff
@@ -512,11 +512,10 @@ struct tau_tracker
                     fmt::arg("error_code", static_cast<int>(rc))));
             return {};
         }
-        auto const info_uniq = std::unique_ptr<addrinfo, void (*)(addrinfo*)>{ info,
-                                                                               [](addrinfo* p) // MSVC forced my hands
-                                                                               {
-                                                                                   freeaddrinfo(p);
-                                                                               } };
+        auto const info_uniq = std::unique_ptr<addrinfo, void (*)(addrinfo*)>{
+            info,
+            [](addrinfo* p) { freeaddrinfo(p); }, // MSVC forced me to write this lambda wrapper
```

Comment:
Not really a formatting issue, but does MSVC force one to write that lambda if template dtor parameter is specified as `decltype(&freeaddrinfo)`?

Reply:
Wasn't able to reproduce with godbolt using 3 different variants. Anyway, if it works it works, could be simplified later.

Reply:
> Not really a formatting issue, but does MSVC force one to write that lambda if template dtor parameter is specified as `decltype(&freeaddrinfo)`?

Just tried, it does work. 👍🏻 

> Wasn't able to reproduce with godbolt using 3 different variants. Anyway, if it works it works, could be simplified later.

https://godbolt.org/z/jo9br6s88